index.js

```diff
@@ -217,6 +217,7 @@ const prepareContext = ctx => {
    * templates to manipulate the indexed object.
    */
   ctx.byGroup = byGroup(ctx.data);
+  ctx.groupOrder = Object.keys(ctx.groups);
```

Comment:
This is okay, but I think I would have been more inclined to change `byGroup` to return the groups in the order from `ctx.groups` rather than the order they appear in the data.